packages/main/src/TabSeparator.js

```diff
@@ -62,6 +62,10 @@ class TabSeparator extends UI5Element {
 		return this._getTabInStripDomRef;
 	}
 
+	get isOnSelectedTabPath() {
+		return false;
+	}
```

Comment:
Instead of this, try to avoid all places where subTabs are iterated, where isOnSelectedTabPath will be called on this. Not sure, I think this was causing an issue in Tab.hbs, so correct it there by iterating through only the ui5-tab elements.